packages/site/mosaic.config.js

```diff
@@ -9,36 +9,18 @@ const mosaicConfig = require('@jpmorganchase/mosaic-standard-generator/dist/fs.c
  * sources: <...source definitions>
  */
 module.exports = deepmerge(mosaicConfig, {
-  sources: [
-    /**
-     * Demonstrates a local file-system source, in this case a relative path to where the
-     * site was generated.
-     * Access from your browser as http://localhost:3000/local
-     */
-    /**
-     * Demonstrates a remote source, in this case the Mosaic site
-     * Access from your browser as http://localhost:3000/mosaic
-     */
+  plugins: [
     {
-      modulePath: '@jpmorganchase/mosaic-source-git-repo',
-      namespace: 'mosaic', // each site has it's own namespace, think of this as your content's uid
+      modulePath: '@jpmorganchase/mosaic-plugins/BrokenLinksPlugin',
```

Comment:
Not sure if we want to add this to the standard generator or not

Reply:
I think it's fine not to add it. Can people turn off plugins that are added by the standard generator?

Reply:
You can disable standard-generator plugins now 👍 

In the site's config, adding `disabled` to a plugin definition will prevent it being run:

e.g.
```
{
      modulePath: '@jpmorganchase/mosaic-plugins/BrokenLinksPlugin',
      disabled: true
}
```

Reply:
the can set the `disabled` option to disable a plugin but the issue is the site config is merged with this:

```
const mosaicConfig = require('@jpmorganchase/mosaic-standard-generator/dist/fs.config.js');
```

that sort of hides everything from the end user